packages/integrations/vercel/src/edge/adapter.ts

```diff
@@ -0,0 +1 @@
+import './throw.js';
```

Comment:
nit: you can remove both of these and just update the package.json imports mapping to point both to `throw.js`